Jenkinsfile

```diff
@@ -76,18 +76,35 @@ def cleanup_master() {
 	}
 }
 
+def archive_logs() {
+	sh '''
+	# need a unique and descriptive name so that the artifacts don't clobber each other
+	# and we can know which log file goes to which node, etc
+	# this also removes a / that would otherwise be included
+	mv "${WORKSPACE%@*}/logs" "${WORKSPACE}/logs_${NODE_NAME}_${JOB_BASE_NAME}_${BUILD_ID}"
+	'''
+	archiveArtifacts "logs_${NODE_NAME}_${JOB_BASE_NAME}_${BUILD_ID}/*"
+	sh '''
+	cd "$(echo "$WORKSPACE" | cut -f 1 -d '@')"
```

Comment:
Why is this still here? I think the 2 lines above (the `mv` one and the `archiveArtifacts` one) are enough.

Reply:
I'll get rid of it. Having old log directories was a nuisance for me, but in practice, it shouldn't be an issue since those directories should all disappear after PRs get merged